pyproject.toml

```diff
@@ -0,0 +1,98 @@
+[tool.poetry]
+name = "algokit-arc"
+version = "0.0.0"
+description = "AlgoKit ARC Library"
+authors = ["Algorand Foundation <contact@algorand.foundation>"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+beaker-pyteal = "^0.4.2"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.2.0"
+black = {extras = ["d"], version = "^22.10.0"}
+ruff = "^0.0.191"
+pip-audit = "^2.4.7"
+pytest-mock = "^3.10.0"
+mypy = "^0.991"
+python-semantic-release = "^7.32.2"
+pytest-cov = "^4.0.0"
+pre-commit = "^2.20.0"
+markdown = "^3.4.1"
+bs4 = "^0.0.1"
```

Comment:
What is this?

Reply:
It is used in this test to inspect the documentation for some basic links https://github.com/algorandfoundation/algokit-arc-library/pull/1/files#diff-15dfeaba21e5e85d8deed84ee029e3d47fae46232e7b857d12ed8a66d9cb5bdeR16